AGENTS.md

```
@@ -91,6 +92,59 @@ npm run build
+# One suite only (useful for triage)
+REVISIUM_CLI_PACKAGE=revisium@2.5.0-alpha.0 npm run test:e2e:matrix -- --testPathPattern=M06
+
+# Stream standalone logs to stderr (debug startup or seeding issues)
+E2E_STANDALONE_LOGS=1 REVISIUM_CLI_PACKAGE=revisium@2.5.0-alpha.0 npm run test:e2e:matrix
+```
+
+### Prompt template for triaging an alpha matrix run
+
+When a suite fails against `revisium@<alpha>`, paste this into your assistant:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)